doc/make_external_gallery.py

```diff
@@ -150,6 +150,24 @@ def format(self):
         link="https://github.com/JacobBumgarner/grad-descent-visualizer",
         image="grad_descent_visualizer.gif",
     ),
+    nikolov1=Example(
+        title="Ivan Nikolov on Visualization Libraries",
+        description="Python Libraries for Mesh, Point Cloud, and Data Visualization (Part 1)",
+        link="https://medium.com/towards-data-science/python-libraries-for-mesh-and-point-cloud-visualization-part-1-daa2af36de30",
+        image="nikolov1.gif",
+    ),
+    nikolov2=Example(
+        title="Ivan Nikolov on Voxelization and Point Clouds",
```

Comment:
Sorry, I promise this is the last comment for a while :D I misread the original title: it's "voxelize (meshes and point clouds)". So we can just use
```suggestion
        title="Ivan Nikolov on Voxelization",
```

Reply:
You can make as many comments as you like.